tests/test_iterable_dataset.py

```diff
@@ -502,6 +502,42 @@ def test_filtered_examples_iterable_with_indices(n, func, batched, batch_size):
     assert [x for _, x in ex_iterable] == expected
 
 
+@pytest.mark.parametrize(
+    "n, func, batched, batch_size, fn_kwargs",
+    [
+        (3, lambda x, y: x["id"] == y, False, None, {"y": 1}),  # keep even number
+        (25, lambda x, y: [i == y for i in x["id"]], True, 10, {"y": 1}),  # same with bs=10
+        (5, lambda x, y: [i == 0 for i in x["id"]], True, None, {"y": 1}),  # same with bs=None
+        (5, lambda x, y: [i == 0 for i in x["id"]], True, -1, {"y": 1}),  # same with bs<=0
+    ],
+)
+def test_filtered_examples_iterable_with_fn_kwargs(n, func, batched, batch_size, fn_kwargs):
```

Comment:
This test seems redundant with the previous ones, I think you can remove it and we can keep the other tests you added

Reply:
I have addressed your comments. Please confirm that it is as intended.
9bd41d4dc2c1b544fa3efa8c7b1308340dbc5835

Reply:
Sorry I meant you can remove `test_filtered_examples_iterable_with_fn_kwargs` completely